### PR TITLE
Fix spree routes

### PR DESCRIPTION
**Pull request by jtapia**
Issue: https://github.com/solidusio/solidus_auth_devise/issues/100

This PR brings the solution mentioned on that issue

### CODE REVIEW

**Comment by jtapia** [comment]
@tvdeyen done, I removed the unnecessary changes, let me know if need any other change

File: spec/support/database_cleaner.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+require 'database_cleaner'
```
Comment (spaghetticode) [inline spec/support/database_cleaner.rb]: @jtapia is this file required? It's not clear to me the reason why it was added, as the commit message only states `Add database_cleaner configuration`. 

Can you change the commit message to include an explanation? That would be great for documentation purposes.

Comment (jtapia) [reply spec/support/database_cleaner.rb]: I added this configuration file because the gem is required but never used, here https://github.com/solidusio/solidus_auth_devise/blob/master/solidus_auth_devise.gemspec#L31

**Comment by spaghetticode** [comment]
This PR seems to fix also #24

File: spec/support/database_cleaner.rb (hunk 1 above)
Comment (jtapia) [reply spec/support/database_cleaner.rb]: ok, I just removed the commit that included it

**Comment by spaghetticode** [comment]
@tvdeyen can you please review after the recent the changes?
